docs/docs/coverage/language/golang.md

```diff
@@ -22,9 +22,9 @@ The table below provides an outline of the features Trivy offers.
 | Binaries |      ✅      | Exclude          |                  -                   | ✅[^4]  |                Not needed                |
 
 !!! note
-    Trivy scans only dependencies of the Go project.
-    Let's say you scan the Docker binary, Trivy doesn't detect vulnerabilities of Docker itself.
-    Also, when you scan go.mod in Kubernetes, the Kubernetes vulnerabilities will not be found.
+    When scanning Go projects, Trivy scans only dependencies of the project, and will not detect vulnerabilities of application itself. 
+    For example, when scanning the Docker binary, Trivy might find vulnerabilities in Go modules that Docker depends on, but won't find 
+    vulnerabilities of Docker itself. Similarly, when scanning the code of Kubernetes, Kubernetes vulnerabilities will not be found.
```

Comment:
ok, I will add a clarification, but aside, do we need this comment here? WDYT about removing the line about k8s.

Reply:
I think we should keep this line.
K8s line shows that we don't find vulnerabilities for `module` line from `go.mod`.
For this case:
We don't find vulnerabilities for `k8s.io/kubernetes` - https://github.com/kubernetes/kubernetes/blob/abc056843c030ead11551e13171a4447701db24e/go.mod#L7)

But maybe we can choose another repository to avoid confusion.

Reply:
We say about docker as a Go binary.
I think it's better to show users that this nuance applies to both `go binary` and `go.mod` files.

Reply:
ok, I hope I clarified it [here](https://github.com/aquasecurity/trivy/pull/7442/commits/e7089702cfb2abc473008bbc6d01dee69ea5a025), if this isn't what you meant please don't hasitate to say

Reply:
I think it looks good.